### ISSUE TITLE
Error with meteor 1.3-rc.8

### ISSUE THREAD

**Issue by lyquocnam** [thereactivestack/meteor-webpack#120]
```
W20160323-09:48:19.325(7)? (STDERR)
W20160323-09:48:19.325(7)? (STDERR) fs.js:696
W20160323-09:48:19.325(7)? (STDERR)   return binding.stat(pathModule._makeLong(path));
W20160323-09:48:19.325(7)? (STDERR)                  ^
W20160323-09:48:19.325(7)? (STDERR) Error: ENOENT, no such file or directory 'D:\D\DEV\Meteor\WEBPACK\rc8\node_modules' W20160323-09:48:19.325(7)? (STDERR)     at Object.fs.statSync (fs.js:696:18)
W20160323-09:48:19.325(7)? (STDERR)     at findAppDir (D:\DEV\Meteor\WEBPACK\rc8\.meteor\local\build\programs\server\npm-require.js:9:10)
W20160323-09:48:19.325(7)? (STDERR)     at addByPath (D:\DEV\Meteor\WEBPACK\rc8\.meteor\local\build\programs\server\npm-require.js:58:22)
W20160323-09:48:19.325(7)? (STDERR)     at registerNodeModules (D:\DEV\Meteor\WEBPACK\rc8\.meteor\local\build\programs\server\npm-require.js:45:5)
W20160323-09:48:19.325(7)? (STDERR)     at D:\DEV\Meteor\WEBPACK\rc8\.meteor\local\build\programs\server\npm-require.js:37:9
W20160323-09:48:19.325(7)? (STDERR)     at Array.forEach (native)
W20160323-09:48:19.325(7)? (STDERR)     at Function._.each._.forEach (C:\Users\lyquo\AppData\Local\.meteor\packages\meteor-tool\1.2.5-rc.8\mt-os.windows.x86_32\dev_bundle\server-lib\node_modules\underscore\underscore.js:79:11)
W20160323-09:48:19.325(7)? (STDERR)     at Object.<anonymous> (D:\DEV\Meteor\WEBPACK\rc8\.meteor\local\build\programs\server\npm-require.js:30:3)
W20160323-09:48:19.325(7)? (STDERR)     at Module._compile (module.js:456:26)
W20160323-09:48:19.325(7)? (STDERR)     at Object.Module._extensions..js (module.js:474:10)
=> Exited with code: 8
W20160323-09:48:35.065(7)? (STDERR)
W20160323-09:48:35.065(7)? (STDERR) fs.js:696
W20160323-09:48:35.065(7)? (STDERR)   return binding.stat(pathModule._makeLong(path));
W20160323-09:48:35.065(7)? (STDERR)                  ^
W20160323-09:48:35.065(7)? (STDERR) Error: ENOENT, no such file or directory 'D:\D\DEV\Meteor\WEBPACK\rc8\node_modules' W20160323-09:48:35.065(7)? (STDERR)     at Object.fs.statSync (fs.js:696:18)
W20160323-09:48:35.065(7)? (STDERR)     at findAppDir (D:\DEV\Meteor\WEBPACK\rc8\.meteor\local\build\programs\server\npm-require.js:9:10)
W20160323-09:48:35.065(7)? (STDERR)     at addByPath (D:\DEV\Meteor\WEBPACK\rc8\.meteor\local\build\programs\server\npm-require.js:58:22)
W20160323-09:48:35.065(7)? (STDERR)     at registerNodeModules (D:\DEV\Meteor\WEBPACK\rc8\.meteor\local\build\programs\server\npm-require.js:45:5)
W20160323-09:48:35.065(7)? (STDERR)     at D:\DEV\Meteor\WEBPACK\rc8\.meteor\local\build\programs\server\npm-require.js:37:9
W20160323-09:48:35.065(7)? (STDERR)     at Array.forEach (native)
W20160323-09:48:35.065(7)? (STDERR)     at Function._.each._.forEach (C:\Users\lyquo\AppData\Local\.meteor\packages\meteor-tool\1.2.5-rc.8\mt-os.windows.x86_32\dev_bundle\server-lib\node_modules\underscore\underscore.js:79:11)
W20160323-09:48:35.065(7)? (STDERR)     at Object.<anonymous> (D:\DEV\Meteor\WEBPACK\rc8\.meteor\local\build\programs\server\npm-require.js:30:3)
W20160323-09:48:35.065(7)? (STDERR)     at Module._compile (module.js:456:26)
W20160323-09:48:35.065(7)? (STDERR)     at Object.Module._extensions..js (module.js:474:10)
=> Exited with code: 8
=> Your application is crashing. Waiting for file change.
```

**Comment by eXon** [comment]
Have you run your `npm install`? It seems like your node_modules folder is missing

**Comment by lyquocnam** [comment]
all packages installed but still errors @eXon

**Comment by eXon** [comment]
Oh! Looks like a Meteor issue, we'll follow that

**Comment by lyquocnam** [comment]
this fixed on meteor 1.3 rc.10
